### DISCUSSION TITLE
Locked Funds

### DISCUSSION THREAD

**Discussion by IgbozeIsrael** [near-daos/astro-ui#1171] [Q&A]
How do we get funds unlocked in the Treasury? Funds were sent by MarketingDAO for guild operations

**Comment by starpause** [comment]
Can you provide a link to your DAO? 

It's most likely that the funds are not in a lockup contract and only require you to create a payout proposal to access them. See this guide:

https://marmaj.org/dao-guide/

**Comment by IgbozeIsrael** [comment]
Hello!
Issue has been resolved as you suggested. Thanks for reaching out.
Best.

Igboze
